pylint/config/callback_actions.py

```diff
@@ -283,3 +284,63 @@ def __call__(
         print(self.run.linter.help())
 
         sys.exit(0)
+
+
+class _AccessLinterObjectAction(_CallbackAction):
+    """Action that has access to the Linter object."""
+
+    def __init__(
+        self,
+        option_strings: Sequence[str],
+        dest: str,
+        nargs: None = None,
+        const: None = None,
+        default: None = None,
+        type: None = None,
+        choices: None = None,
+        required: bool = False,
+        help: str = "",
+        metavar: str = "",
+        **kwargs: "PyLinter",
```

Comment:
I don't understand this one, we want to be able to handle any additional keyword argument and dump them ?

Reply:
No, it is similar to the `Run` actions. In the option dictionary I create the following key: `kwargs: {"linter": self}`. This means that by accessing `kwargs["linter"]` and setting it on the `Action` as instance attribute I can access the linter in `__call__`.
By adding this typing here we add typing to `self.linter` for this action.